spinalcordtoolbox/image.py

```diff
@@ -910,10 +910,10 @@ def concat_data(im_in_list: Sequence[Image], dim, pixdim=None, squeeze_data=Fals
         data_concat = np.concatenate(dat_list, axis=dim)
 
     # write file
-    fname_out = im_in_list[0].absolutepath
```

Comment:
since `im_in_list[0]` seems to be used more as an input image rather than a list, might be clear to do something like: `im_in = im_in_list[0]` at the beginning of this code block

Reply:
Good point about readability! 

We use `im_in_list[0]` because we need a header and path for the output image, but we have multiple input images to choose from, so we pick the first image. This could definitely be made more clear.